interface/SimpleCacheSentry.h

```diff
@@ -8,12 +8,14 @@
 class SimpleCacheSentry : public RooAbsArg {
     public:
         SimpleCacheSentry() ;
+        SimpleCacheSentry(const char *name, const char *title) ;
```

Comment:
For optimisation purposes there are places where sentries are added as dependents of other sentries, but for this to work the sentry object needs to have a unique name.